### DISCUSSION TITLE
Can I use a XPT2046 with a ILI9488 display?

### DISCUSSION THREAD

**Discussion by Frtrillo** [ImpulseAdventure/GUIslice#446] [Q&A]
Hello, im a first timer using this library and im going to test components to use it.
Ive seen this display combination of XTP2046 with ILI9488 of a provider, but I haven't seen that combination in the /configs folder, only with the 9341

**Comment by Pconti31** [comment]
@Frtrillo I don't  believe a pre-packaged config file exists.  The owner of this repository  @ImpulseAdventure (Calvin) would know.  Nevertheless config files are not hard to create.  

The first issue is to get your display without touch going.  This requires you to choose a TFT driver that GUIslice can use, say the one provided by [jaretburkett/ILI9488](https://github.com/jaretburkett/ILI9488).  If you are using a micro controller like Arduino use the config file  `ard-adagfx-ili9488-notouch.h`.  Get that working and touch can follow on.

You should also be aware the ILI9488 display can be really slow  with 8 bit micro controllers like the Arduino Mega.  Although with some software hacking  jaretburkett/ILI9488 driver can be made faster.  

So what are you using for a controller?   

Paul--

**Reply by Frtrillo** [reply]
Oh also I forgot answer your question Ill use an esp32 as controller

**Comment by Pconti31** [comment]
@Frtrillo Also, if you haven't actually purchased this TFT display yet I would avoid it and pick a faster and supported one.  
Paul--

**Comment by Pconti31** [comment]
@Frtrillo Crap, one other thing I forgot to mention. These display often comes with two different interfaces, 4-wire spi and 8 bit parallel.  My answer above is for 4-wire spi.  If yours is 8 bit parallel you will need to use the MCU Friend library and one of its config files. 

Often with the 8 bit parallel version while its advertised as ili9488 most of the time it's whatever spare IC the manufacture has laying around.  The display is still slow though with this interface plus you will have very few pins left over to do anything with.  Ugh. Not a great display, thus the very low prices.
Paul--

**Reply by Frtrillo** [reply]
Alright thanks for all your answers ! Indeed I'm still looking for touch displays on the cheaper side at Alibaba, suggested that because it was the first I found with a supported touch controller hahah
I'm open for suggestions as well

**Comment by Pconti31** [comment]
@Frtrillo One thing you can do is checkout the wiki page:
[Display-Config-Table](https://github.com/ImpulseAdventure/GUIslice/wiki/Display-Config-Table)
Paul--
